### ISSUE TITLE
Probleme loading group Magic Fields 1.7.3 / WordPress 5.0.3

### ISSUE THREAD

**Issue by parmarhardip** [hunk/Magic-Fields#197]
I have updated the latest version of WordPress but backend side not showing the data of a magic field.

So Kindly, Check the Magic fields plugin is compatible with latest WordPress

**Comment by johncrumpton** [comment]
Yes, I'm getting a white page on the edit page. Installing the _Classic Editor_ plugin resolves this.
